mk_verificator/tests/service_checker/config.yaml

```diff
@@ -1,4 +1,4 @@
 ---
-skipped_nodes: ['saml-global-01.mosci.jiocloud.com']
+skipped_nodes: []
 skipped_group: []
-skip_number: 1
+skip_number: 0
```

Comment:
that doesn't seem as sensitive information, also I would recommend to add comment for this option (what it is and for which purpose user has to specify it)